x-pack/plugins/security_solution/server/endpoint/routes/resolver/entity/utils/supported_schemas.ts

```diff
@@ -64,15 +74,15 @@ export const supportedSchemas: SupportedSchema[] = [
     },
   },
   {
-    name: 'sysmonViaFilebeat',
+    name: 'filebeat',
```

Comment:
I'm sure there is a reason to change this but I'm a bit lost on this context. Could you elaborate a little bit about why is this change needed? Thanks!

Reply:
Yeah my understanding was that this name is not being used anywhere. But my change may affect some user's filters or something so I'll do it differently. Thanks for bringing my attention to this :)